### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by zakrzaq**

### CODE REVIEW

File: src/news_app/news_app.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { makeFeed } from './create_feed';
+import { getData } from './get_data';
+import { clearFeed } from './clear_feed';
+// import { updateFeed } from './update_feed.js';
+
+import './news_style.css';
+
+// VARIABLES
+const urlFood = `https://api.thenewsapi.com/v1/news/all?api_token=NfHTFOIIekZazvqWjUldhd6FX72AKBkguV1ye0D2&categories=food&language=en`;
```
Comment (radekwojpl2) [inline src/news_app/news_app.js:9]: create a variable for a toke and use string interpolation, same with URL and categories

File: src/news_app/news_style.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.news-list {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 30px;
```
Comment (radekwojpl2) [inline src/news_app/news_style.css:11]: i think you should use ems and rems units

File: src/style.css [hunk 3]
```diff
@@ -1,8 +1,92 @@
...
-  margin-top: 60px;
+  margin-top: 0px;
+}
+
+:root {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
```
Comment (radekwojpl2) [inline src/style.css:17]: i think those *::before and *::after can be simplify to ::before and ::after, sorry my bad. check that out

File: src/views/ShoppingList/shoppingList.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import main from '../../main.js'
+console.log("test")
```
Comment (radekwojpl2) [inline src/views/ShoppingList/shoppingList.js:2]: delete this file

File: vite.config.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+const { resolve } = require('path')
+const { defineConfig } = require('vite')
+
+console.log(__dirname)
```
Comment (radekwojpl2) [inline vite.config.js:4]: delete this line

File: src/news_app/news_style.css (hunk 2 above)
Comment (radekwojpl2) [reply src/news_app/news_style.css:11]: please read about them

File: src/news_app/news_app.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import { makeFeed } from './create_feed';
+import { getData } from './get_data';
+import { clearFeed } from './clear_feed';
+// import { updateFeed } from './update_feed.js';
+
+import './news_style.css';
```
Comment (radekwojpl2) [inline src/news_app/news_app.js:6]: do we need this here ?

File: src/news_app/create_feed.js [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    }
+    // ADD INFO SECTION
+    const infoContainer = document.createElement('div');
+    infoContainer.classList.add('info-container');
+    // ADD TITLE HEADER
+    const newsHead = document.createElement('h3');
+    newsHead.innerText = title;
+    infoContainer.appendChild(newsHead);
+    // // ADD AUTHOR
+    // if (author) {
```
Comment (radekwojpl2) [inline src/news_app/create_feed.js:57]: clear that

File: src/componenets/header.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    super();
+
+    this.links = [
+      {
+        name: 'HOME',
+        href: 'index.html',
+        active: this.checkPathname('index.html'),
+      },
+      {
+        name: 'SHOPPING LIST',
```
Comment (radekwojpl2) [inline src/componenets/header.js:12]: comments out those elements which we are not using right now

File: src/componenets/header.js [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+        name: 'FAST FOOD',
+        href: 'page.html',
+        active: this.checkPathname('p.html'),
+      },
+    ];
+  }
+
+  checkPathname(path) {
+    const pathname = window.location.pathname.split('/');
+    return pathname[1] === path;
```
Comment (radekwojpl2) [inline src/componenets/header.js:36]: magic number, variable with some good name would be better, when you get back to this code you will not have to think what this 1 is

File: index.html [hunk 10]
```diff
@@ -4,13 +4,34 @@
 <head>
```
Comment (radekwojpl2) [inline index.html:4]: You did not updated readme file, please do so

File: index.html [hunk 11]
```diff
@@ -4,13 +4,34 @@
...
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <link rel="stylesheet" href="./src/style.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./src/style.css">
   <title>Vite App</title>
```
Comment (radekwojpl2) [inline index.html:10]: title can be change too

File: index.html [hunk 12]
```diff
@@ -4,13 +4,34 @@
...
 </head>
 
 <body>
-  <div id="app"></div>
-  <script type="module" src="./src/main.js"></script>
+
+  <div id="app">
+    <component-header></component-header>
+    <main>
+      <div class="news">
```
Comment (radekwojpl2) [inline index.html:18]: please read about semantic html

File: src/news_app/news_app.js (hunk 6 above)
Comment (zakrzaq) [reply src/news_app/news_app.js:6]: Styles? I was trying to keep this modules style out of the main css - which can become long very quick

Comment (zakrzaq) [reply src/news_app/news_app.js:6]: I can transfer them over if required

**Pull request merged**